syntax="proto2";

message LoggerEvent
{
    enum Event
    {
	ALL_LOGS_CLOSED_FOR_PACKET = 1;
	PACKET_START = 2;
    }
    required Event event = 1;
    
    optional string log_dir = 2;
    optional string start_time = 3;
    required int32 packet_id = 4;

    optional int32 tx_modem_id = 5;    
}
